### PR TITLE
fix(a11y): reset height for reduced motion #42

### LINKED ISSUE #42: Height not reset with 'prefers-reduced-motion' enabled
## Describe the bug
When prefers-reduced-motion is enabled, the height added to the accordion panel is not removed after the transition completes.

image

This causes issues when there is dynamic content inside of the accordion. For example, one of our pages has form fields that are added/removed inside an accordion based on a radio button selection. When reduced-motion is enabled, any newly added fields are cut off since the accordion still has a height set.

## To Reproduce
1. Enable prefers-reduced-motion
2. Go to the [accordion demo](https://auro.alaskaair.com/components/auro/accordion)
3. Expand the accordion. Notice that height is not removed.

## Expected behavior
The height does not persist when reduced-motion is enabled.

## Desktop (please complete the following information):
 - OS: Windows 10
 - Browser: Chrome
 - Version: 1.4.0

### PR DESCRIPTION
**Pull request by geoffrich**
# Alaska Airlines Pull Request

Fixes #42

## Summary:

Update WCSS dependency with changes from https://github.com/AlaskaAirlines/WebCoreStyleSheets/pull/111. This correctly resets the accordion height on open/close when reduced motion is turned on.

I made the commit type "fix" instead of "chore" because the dependency update is fixing a real issue, and wanted to trigger a release.

@blackfalcon updating the dep also updated the package lock. Since I'm on the latest npm, it also updated the lockfile format to be v2 which changed 20k+ lines. Is that okay, or do you want me to handle it differently?

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,15 +24,15 @@
   },
   "peerDependencies": {
     "@alaskaairux/design-tokens": "^3.0.0",
-    "@alaskaairux/webcorestylesheets": "^3.0.0",
     "@alaskaairux/orion-icons": "^2.1.5",
+    "@alaskaairux/webcorestylesheets": "^3.0.0",
```
Comment (blackfalcon) [inline package.json:28]:
```suggestion
    ""@alaskaairux/webcorestylesheets": "^3.7.3",
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (geoffrich) [reply package.json:28]: That's the peer-dep -- I don't think that should be the same version requirements?

This bug is internal to the package and will be fixed regardless of whether the user has fully updated WCSS. They don't need to be on the latest WCSS to use the package - any v3 should be fine. Or am I missing something?

Comment (blackfalcon) [reply package.json:28]: Wait... why is the peer dependency even ther? It's not needed to run the element? All the Sass is processed into the element itself, so there is no dependency other than DEV. 

So, please remove that.

Comment (geoffrich) [reply package.json]: Removed.

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**